v0/pipeline/steps/iacm/iacmterragrunt-plugin-info.yaml

```diff
@@ -0,0 +1,50 @@
+title: IACMTerragruntPluginInfo
+allOf:
+  - $ref: ../../common/step-spec-type.yaml
+  - type: object
+    required:
+      - command
+    properties:
+      command:
+        type: string
+      image:
+        type: string
+      resources:
+        $ref: ../../common/container-resource.yaml
+      target:
+        type: array
+        items:
+          type: string
```

Comment:
If you want to do array fields runtime too, you need to add it as:

oneOf:
          - type: array
            items:
              type: string
          - type: string
            pattern: ^<\+input>((\.)((executionInput\(\))|(allowedValues|default|regex)\(.+?\)))*$
            minLength: 1

Reply:
changed.